state-chain/runtime/src/lib.rs

```diff
@@ -1708,6 +1709,23 @@ impl_runtime_apis! {
 		fn cf_account_flip_balance(account_id: &AccountId) -> u128 {
 			pallet_cf_flip::Account::<Runtime>::get(account_id).total()
 		}
+		fn cf_common_account_info(
+			account_id: &AccountId,
+		) -> RpcAccountInfoCommonItems<FlipBalance> {
+			LiquidityPools::sweep(account_id).unwrap();
+			let flip_account = pallet_cf_flip::Account::<Runtime>::get(account_id);
+			RpcAccountInfoCommonItems {
+				flip_balance: flip_account.total(),
+				asset_balances: AssetBalances::free_balances(account_id),
+				bond: flip_account.bond(),
+				estimated_redeemable_balance: pallet_cf_funding::Redemption::<Runtime>::for_rpc(
+					account_id,
+				).map(|redemption| redemption.redeem_amount).unwrap_or_default(),
+				bound_redeem_address: pallet_cf_funding::BoundRedeemAddress::<Runtime>::get(account_id),
+				restricted_balances: pallet_cf_funding::RestrictedBalances::<Runtime>::get(account_id),
+				delegating_to: pallet_cf_validator::DelegationChoice::<Runtime>::get(account_id),
```

Comment:
this is just their current choice for operator right? if they are switching in the middle of an epoch, this will be the new operator they've chosen and not the one they are "bonded" with

Reply:
Yes, this is the current choice.